rails_app/app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,23 @@
+%html
+  %head
+    %title
+      Price monitor
+
+    = stylesheet_link_tag 'application'
+    = javascript_include_tag 'application'
+    = csrf_meta_tags
+
+
+  %body
+    = render 'layouts/header'
+    - flash.each do |message_type, message|
+      .alert{ class: "alert-#{message_type}" }
```

Comment:
Not sure what this alert does

Reply:
http://api.rubyonrails.org/classes/ActionDispatch/Flash.html